packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -185,6 +185,18 @@ export interface IFluidContainer<TContainerSchema extends ContainerSchema = Cont
 	 */
 	attach(props?: ContainerAttachProps): Promise<string>;
 
+	/**
+	 * Capture the state of a container that is not attached or closed.
+	 *
+	 * @remarks
+	 *
+	 * This should only be called when the container is in the
```

Comment:
This doesn't appear to be enforced by the implementation. What is the expected outcome if this is called in non-detached states?

Reply:
Good callout, forgot to do this.